src/platform/user/profile/actions/index.js

```diff
@@ -44,8 +44,12 @@ export function refreshProfile(
   };
   return async dispatch => {
     const url = forceCacheClear ? appendQuery(baseUrl, query) : baseUrl;
-
     const payload = await apiRequest(url);
+    const { organicModal } = await apiRequest(
+      '/user_transition_availabilities',
+    );
+    payload.data.attributes.showOrganicAdoptionExperimentModal = organicModal;
```

Comment:
This is fine for now I guess but this should really be bound to the `/v0/user` (if BE has a problem just tell them I said  to do it).

My reasoning for this is because it'll create duplicate Network calls after every call to the `/refresh` data that we already have upon first successful authentication.

Reply:
Yes, that's what I wanted to happen and as you predicted, BE folks were against it.

Reply:
@bosawt what are your thoughts on this? I prefer this be bound to user so we dont have the extra calls, but wanted to check with you

Reply:
@kscoulter Another option is to put this in `user-nav/containers/Main.jsx` and put it in the `componentDidUpdate` in the `currentlyLoggedIn` if statement (lines 68 - 87)